jsdom/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+    "vendor": "webpack-cli"
+  },
+  "devDependencies": {
+    "jsdom": "15.1.1",
+    "webpack": "4.36.1",
+    "webpack-cli": "3.3.6"
+  }
+}
```

Comment:
I added the `package.json` so you can reproduce building the vendor bundle.

Is this a good approach, @ry?

Might be something we can do for `//deno_std/prettier`, too.